### PR TITLE
Address validation

### LINKED ISSUE #25: address validation when sending funds

### PR DESCRIPTION
**Pull request by severinwullschleger**
closes #25

### CODE REVIEW

File: src/ui/core/Input.tsx [hunk 1]
```diff
@@ -34,6 +34,7 @@ const StyledInput = styled(TextInput)`
   flex: 1;
   color: ${__COLORS.WHITE};
   padding: ${SPACING * 2}px 0;
+  padding-right: ${(props) => (props.status === 'valid' ? 20 : 0)}px
```
Comment (severinwullschleger) [inline src/ui/core/Input.tsx:37]: @JonnyBurger how can i solve this linting error?

Comment (JonnyBurger) [reply src/ui/core/Input.tsx:37]: @severinwullschleger Add a type:


```
const StyledInput = styled(TextInput)<{status: string}>``
```

Also here should be a semi colon

```suggestion
  padding-right: ${(props) => (props.status === 'valid' ? 20 : 0)}px;
```

**Comment by JonnyBurger** [comment]
Sorry closed wrong one

File: src/ui/core/Input.tsx [hunk 2]
```diff
@@ -130,6 +131,7 @@ export const Input = ({
         onChangeText={onChangeText}
         onBlur={onblur}
         onFocus={onfocus}
+        status={status}
```
Comment (severinwullschleger) [inline src/ui/core/Input.tsx:134]: how do I have to solve this one

**Pull request merged**